src/CarSearch/CarSearchInfoBox.js

```diff
@@ -0,0 +1,31 @@
+/* global google */
+
+import InfoBox from 'google-maps-infobox';
+import './styles/CarSearchInfoBox.css';
+
+export function createInfoBox(location) {
+  let content = '<div class="CarSearchInfoBox">' +
+    '<div class="iw-top"></div>' +
+    '<div class="iw-mdl">';
+
+  if (location.cars.length <= 1) {
+    const car = location.cars[0];
+    content += `<p class="car">${car.model} <span class="type ${car.type}"></span>`
```

Comment:
is it possible to make this a component as well. This way we can use it like other components instead of concat'ing raw html. We can then create an outward function within the component that simply returns the rendered jsx, because maps of course still needs raw html. Just a thought